packer/plugin_ext_unix.go

```diff
@@ -0,0 +1,9 @@
+// +build aix darwin dragonfly freebsd js,wasm linux netbsd openbsd solaris
+
+package packer
+
+import pluginsdk "github.com/hashicorp/packer-plugin-sdk/plugin"
+
+var (
+	PluginFileExtension = "_x" + pluginsdk.APIVersion // OS-Specific plugin file extention
```

Comment:
I might have missed this. But is the API version required in the filename?

Reply:
Yes

Reply:
( Terraform does the same )